ros2bag/ros2bag/verb/play.py

```diff
@@ -182,8 +200,23 @@ def main(self, *, args):  # noqa: D102
         play_options.node_prefix = NODE_NAME_PREFIX
         play_options.rate = args.rate
         play_options.topics_to_filter = args.topics
-        play_options.topics_regex_to_filter = args.regex
-        play_options.topics_regex_to_exclude = args.exclude
+
+        # Convert service name to service event topic name
+        play_options.services_to_filter = convert_service_to_service_event_topic(args.services)
+
+        play_options.regex_to_filter = args.regex
+
+        if args.exclude:
+            print(print_warn(str('-x/--exclude argument is deprecated. Please use '
+                                 '--exclude-topics.')))
+            play_options.topics_regex_to_exclude = args.exclude
+        else:
+            play_options.topics_regex_to_exclude = args.exclude_topics
+
+        if args.exclude_services:
+            play_options.services_regex_to_exclude = args.exclude_services + '/_service_event'
```

Comment:
@MichaelOrlov 

Sorry. I think current description of the below parameter is wrong. For regular expression, it should not use more than one. So space isn't needed.  
```
  -e REGEX, --regex REGEX
                        filter topics by regular expression to replay, separated by space. If none specified, all topics will be replayed.
  -x EXCLUDE, --exclude EXCLUDE
                        regular expressions to exclude topics from replay, separated by space. If none specified, all topics will be replayed. This argument is deprecated and please use --exclude-topics.
  --exclude-topics EXCLUDE_TOPICS
                        regular expressions to exclude topics from replay, separated by space. If none specified, all topics will be replayed.
  --exclude-services EXCLUDE_SERVICES
                        regular expressions to exclude services from replay, separated by space. If none specified, all services will be replayed.
```
Besides, I think we should keep the same style as record command.    
 --exclude  => --exclude-regex   
--exclude-topics regular expression => --exclude-topics topics list  
--exclude-services regular expression =>  --exclude-services  services list  
What do you think ?

Reply:
@Barry-Xu-2018 As regards 
> Besides, I think we should keep the same style as record command.
--exclude => --exclude-regex
--exclude-topics regular expression => --exclude-topics topics list
--exclude-services regular expression => --exclude-services services list
What do you think ?

Agree, that it would be great to be consistent with recording parameters and will be also more logically correct.
However, It would be better to move out such unrelated changes from this PR and put them in a separate PR. 
Perhaps we would be able to merge it quickly and rebase this PR with the service playback above it.
If we would separate those changes and merge them before this PR, likely we will be able to backport those changes to the Iron branch.

Reply:
@MichaelOrlov 

The original parameter for play is    
-x EXCLUDE, --exclude EXCLUDE

The modification to the original parameter is from `--exclude`  to `--exclude-regex`. Do you means this change is made as a PR and backport to Iron branch ?  

--exclude-topics regular expression => --exclude-topics topics list  
--exclude-services regular expression => --exclude-services services list  
The above 2 new parameters were added at this PR. So they are related to this PR.

Reply:
@Barry-Xu-2018 As regards
> The modification to the original parameter is from --exclude to --exclude-regex. Do you means this change is made as a PR and backport to Iron branch ?

Up to you. IMO it is not very important since Iron will reach EOL in 7 months i.e. November 1 and not too much users on Iron. People mostly use either Humble or Rolling.